lib/mayaUsd/render/vp2RenderDelegate/proxyRenderDelegate.cpp

```diff
@@ -120,6 +151,57 @@ TfToken GetSelectionKind()
     return TfToken();
 }
 
+// clang-format off
+TF_DEFINE_PRIVATE_TOKENS(
+    _pointInstancesPickModeTokens,
+
+    (PointInstancer)
+    (Instances)
+    (Prototypes)
+);
+// clang-format on
+
+UsdPointInstancesPickMode GetPointInstancesPickMode()
```

Comment:
Please add doc for the method.

Reply:
Addressed in ac6bf39fd219df491dd995fdb6eb0367c97ca46f.